tonic/src/codec/decode.rs

```diff
@@ -156,19 +173,13 @@ impl<T> Streaming<T> {
 
     fn decode_chunk(&mut self) -> Result<Option<T>, Status> {
         if let State::ReadHeader = self.state {
-            if self.buf.remaining() < 5 {
+            if self.buf.remaining() < HEADER_SIZE {
```

Comment:
Look a while to figure out what this hardcoded `5` was. Figured it made sense to extract into a `const`.